railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -16,11 +16,12 @@
 #     # policy.report_uri "/csp-violation-report-endpoint"
 #   end
 #
-#   # Generate session nonces for permitted importmap and inline scripts.
+#   # Generate session nonces for permitted importmap and inline scripts. See:
+#   # https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce
 #   config.content_security_policy_nonce_generator = ->(request) { request.session.id.to_s }
 #   config.content_security_policy_nonce_directives = %w(script-src)
 #
-#   # Report CSP violations to a specified URI. See:
```

Comment:
We could also remove all links to mozilla.org and have a single link to the guides instead.

```suggestion
```

Similar to the `filter_parameter_logging` initializer template:
https://github.com/rails/rails/blob/c01a744674d2edc4a35cc8f5b4cab46a03dbb454/railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb.tt#L4-L5